### PR TITLE
Let the user know if the model is infeasible

### PR DESCRIPTION
**Pull request by kvikshaug**
Depends on https://github.com/DD-DeCaF/simulations/pull/140

### CODE REVIEW

File: src/store/modules/interactiveMap.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export interface Card {
   // General simulation fields
   isSimulating: boolean;
   hasSimulationError: boolean;
+  solverStatus: string | null;
```
Comment (Midnighter) [inline src/store/modules/interactiveMap.ts]: `hasSimulationError` could then just be a computed of `!!solverStatus && solverStatus !== "optimal"`? Maybe not worth all the refactoring.

File: src/views/InteractiveMap/Card.vue [hunk 2]
```diff
@@ -36,6 +36,14 @@
       </v-btn>
     </v-toolbar>
     <v-card-title primary-title class="py-2" v-if="isSelected">
+      <v-alert
+        :value="card.solverStatus && card.solverStatus !== 'optimal'"
```
Comment (Midnighter) [inline src/views/InteractiveMap/Card.vue]: Could this use `hasSimulationError`?

File: src/views/InteractiveMap/Card.vue [hunk 3]
```diff
@@ -259,6 +267,11 @@ export default Vue.extend({
       if (this.isSelected) {
         if (this.card.hasSimulationError) {
           return "error";
+        } else if (
+          this.card.solverStatus &&
```
Comment (Midnighter) [inline src/views/InteractiveMap/Card.vue]: Could this use `hasSimulationError`?

File: src/views/InteractiveMap/Card.vue (hunk 2 above)
Comment (kvikshaug) [reply src/views/InteractiveMap/Card.vue]: Not really, that's used for handling HTTP-level errors like network issues or unhandled backend exceptions.

**Pull request merged**